### DISCUSSION TITLE
What version of OpenZeppelin does Patric use in his advanced foundry course, more specifically in section 3 "DEFI"

### DISCUSSION THREAD

**Discussion by 0xivanalexander** [Cyfrin/foundry-full-course-cu#2778] [Q&A]
I hope you all are having great day!

**Comment by EngrPips** [comment]
Hello @thingsgogood, I can't remember correctly, but I remember he used to show how he installed dependencies. Maybe you can pay attention to when he imports `OpenZeppelin` in the course to see the version tag he used.

**Comment by 0xivanalexander** [comment]
Okay I looked in the first DeFi lessons, and he suggested to look into the Makefile, I totally forgot about that!

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
